client/src/pages/CataloguePage/CataloguePage.js

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Stack, Container, Box, Typography } from '@mui/material';
+import CatalogTourCard from '../../components/CatalogTourCard/CatalogTourCard';
 
-const CataloguePage = () => <div>CATALOGUE</div>;
+const CataloguePage = () => (
+  <Box sx={{ backgroundColor: '#EDEDED' }}>
+    <Container>
+      <Typography sx={{ marginBottom: '40px' }} variant="h2">TOURS</Typography>
+      <Stack spacing={2}>
+        <CatalogTourCard />
+        <CatalogTourCard />
+        <CatalogTourCard />
```

Comment:
this is unpractical in case of receiving hundreds of cards from the backend, please write a more production approach code utilizing the array map method.
you could create a dummy array consisting of card objects with a card header, card text, and card img url and use the map method to display them inside your stack.

Reply:
This will be done later.
This is now presented for illustrative purposes.